config/index.js

```diff
@@ -0,0 +1,12 @@
+const {
+  NODE_ENV,
+  JWT_SECRET_KEY,
+  PORT,
+  SECRET_KEY,
+} = process.env;
+
+module.exports = {
+  PORT: PORT || 3000,
+  JWT_SECRET_KEY: NODE_ENV === 'production' ? JWT_SECRET_KEY : SECRET_KEY,
```

Comment:
Нужно исправить: при запуске локально не работает авторизация.
При запуске локаьно `SECRET_KEY` не объявлен. Вместо того, чтобы для локального запуска передавать значения через ENV, я рекомендую объявить их прямо в коде:
```suggestion
  JWT_SECRET_KEY: NODE_ENV === 'production' ? JWT_SECRET_KEY : 'dev-secret',
```

Лишнюю переменную удалите.